addons/common/functions/fnc_findUnloadPosition.sqf

```diff
@@ -40,12 +40,22 @@ if (_checkVehicleIsStable) then {
 };
 
 private _radiusOfItem = 1;
-if (_typeOfCargo isKindOf "CAManBase") then {
+if (_cargo isKindOf "CAManBase") then {
     _radiusOfItem = 1.1;
 } else {
     //`sizeOf` is unreliable, and does not work with object types that don't exist on map, so estimate size based on cargo size
-    if (isNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size))) then {
-        _radiusOfItem = (((getNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size))) ^ 0.35) max 0.75);
+    private _typeOfCargo = if (_cargo isEqualType "") then {_cargo} else {typeOf _cargo};
+    private _itemSize = if (isNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size))) then {
```

Comment:
this code piece already exists in `EFUNC(cargo,getSizeItem)`, so can be optimized well

Reply:
I did a choice here as explain in the PR description :
> Prefer the config way instead to determine the size of `_cargo` directly with  `ace_cargo_fnc_getSizeItem` (this avoid inconsistency when player `ace_cargo_fnc_setsize` of an object lower from the `ace_cargo_size` defined in config)

I can add a parameter in `EFUNC(cargo,getSizeItem)` to prefer the config if exist or prefer varaible if exist

Reply:
I am supposing that `ace_cargo_size` in config is correct so if I am calling `EFUNC(cargo,getSizeItem)` and if player has `ace_cargo_fnc_setsize` an other value (a smaller one for exemple) the function will return a bad position base on a wrong value changed by player. This avoid that case and also cover the objective of this PR where `ace_cargo_size` in config is not defined.